integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/car/deployment/test/ClassMediatorCarTestCase.java

```diff
@@ -80,129 +51,5 @@ public void capp1DeploymentAndServiceInvocation() throws Exception {
 
     }
 
-    @Test(groups = { "wso2.esb" }, description = "Test Car with Mediator un-deployment", dependsOnMethods = {
-            "capp1DeploymentAndServiceInvocation" })
-    public void capp1UnDeploymentTest() throws Exception {
```

Comment:
could you please add a comment saying why we are removing all these methods.

Reply:
Added. Please refer the latest commit.